### ISSUE TITLE
Add howto for running Cascalog on Elastic MapReduce

### ISSUE THREAD

**Issue by bobpoekert** [Quantisan/cascalog-doc#1]

**Comment by Quantisan** [comment]
which do you think would be more useful? the GUI or command line?

**Comment by bobpoekert** [comment]
I think command line would be easier to reproduce and less likely to break when Amazon changes something. GUI tutorials tend to be brittle and hard to follow.

**Comment by Quantisan** [comment]
@bobpoekert do you have the commands you might have used to run cascalog jar on EMR? if you do, that'd be a good start for this page

**Comment by sorenmacbeth** [comment]
Someone ought to write a tutorial on using my lein-aws plugin to launch
cascalog EMR jobflows _not it_

On Tuesday, December 3, 2013, Paul Lam wrote:

> @bobpoekert https://github.com/bobpoekert do you have the commands you
> might have used to run cascalog jar on EMR? if you do, that'd be a good
> start for this page
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Quantisan/cascalog-doc/issues/1#issuecomment-29752056
> .

## 

http://about.me/soren

**Comment by Quantisan** [comment]
https://github.com/sorenmacbeth/lein-aws/issues/4
